### PR TITLE
Prep for 1.5.6 release

### PR DESCRIPTION
**Pull request by suyograo**
* Lock file update
* release notes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,22 @@
+### 1.5.6 (Dec 8, 2015)
+### general
+  - This release bundles a new version of JRuby - [1.7.23](http://jruby.org/2015/11/24/jruby-1-7-23.html), which fixes a memory leak issue reported on Windows when using the file input ([#3754](https://github.com/elastic/logstash/issues/3754)).
+
+### input
+  - File:
+    - Properly release file handles for older files which allows users to delete them ([#31](https://github.com/logstash-plugins/logstash-input-file/issues/31)).
+    - Clean up resource usage when Logstash is restarted with File input. Previously, this would cause Logstash to grab on to duplicate file handles ([#77](https://github.com/logstash-plugins/logstash-input-file/issues/77)).
```
Comment (jsvd) [inline CHANGELOG.md]: Clean up resource usage when the pipeline restarts an instance of the file input plugin.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,13 +1,22 @@
+### 1.5.6 (Dec 8, 2015)
+### general
+  - This release bundles a new version of JRuby - [1.7.23](http://jruby.org/2015/11/24/jruby-1-7-23.html), which fixes a memory leak issue reported on Windows when using the file input ([#3754](https://github.com/elastic/logstash/issues/3754)).
+
+### input
+  - File:
+    - Properly release file handles for older files which allows users to delete them ([#31](https://github.com/logstash-plugins/logstash-input-file/issues/31)).
```
Comment (jsvd) [inline CHANGELOG.md]: allows users to keep old files in the watched director without having to delete/rename them

**Comment by jsvd** [comment]
LGTM

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,13 +1,22 @@
+### 1.5.6 (Dec 8, 2015)
+### general
+  - This release bundles a new version of JRuby - [1.7.23](http://jruby.org/2015/11/24/jruby-1-7-23.html), which fixes a memory leak issue reported on Windows when using the file input ([#3754](https://github.com/elastic/logstash/issues/3754)).
+
+### input
+  - File:
+    - Properly release file handles for older files which allows users to keep old files in the watched director without having to delete/rename them ([#31](https://github.com/logstash-plugins/logstash-input-file/issues/31)).
```
Comment (jsvd) [inline CHANGELOG.md]: sorry I mistyped "directory" here

Comment (suyograo) [inline CHANGELOG.md]: ahh, i blindly copied it :( will change and merge

**Pull request merged**